hasher-matcher-actioner/hmalib/metrics/query.py

```diff
@@ -64,23 +88,32 @@ def get_count_with_graph(
     """
     Given a time period and a set of metric names, gets the sum of the metric
     over the period and a graphable list of timestamps and values.
+
+    The graph data always contains the start and end time stamps with None values
+    to make graphing easier.
     """
     result = {}
 
+    start_time = _start_time(time_period).replace(second=0, microsecond=0, tzinfo=None)
+    end_time = datetime.now().replace(second=0, microsecond=0, tzinfo=None)
+
     for metric_name in names:
         stats = _get_cloudwatch_client().get_metric_statistics(
             Namespace=METRICS_NAMESPACE,
             MetricName=f"{metric_name}-count",
             Statistics=["Sum"],
-            StartTime=_start_time(time_period),
-            EndTime=datetime.now(),
+            StartTime=start_time,
+            EndTime=end_time,
             Period=_period(time_period),
         )["Datapoints"]
 
         total = int(functools.reduce(lambda acc, s: acc + s["Sum"], stats, 0))
 
-        graph_data = [(s["Timestamp"], int(s["Sum"])) for s in stats]
+        graph_data = [
+            (s["Timestamp"].replace(tzinfo=None), int(s["Sum"])) for s in stats
```

Comment:
Having the browser handle timezone? nit: maybe add a comment explaining that/why the removal

Reply:
Yes. Browser / graphing library handles timezones pretty well.

There is an additional reason, will add as comment.